nmxact/nmble/naked_sesn.go

```diff
@@ -351,6 +351,11 @@ func (s *NakedSesn) TxRxMgmt(m *nmp.NmpMsg,
 	return rsp, nil
 }
 
+func (s *NakedSesn) TxRxMgmt_async(m *nmp.NmpMsg,
+	timeout time.Duration, ch chan nmp.NmpRsp, err_c chan error) error {
+	return nil
```

Comment:
It might be better to return `fmt.Errorf("unsupported")` or similar here.

Reply:
done